### PR TITLE
feat:Guides CosmPy 

### PR DESCRIPTION
**Pull request by NikoDMT**
Contribution to the Getting Started guide for CosmPy. Contribution includes: Installation, Connecting to a chain, Querying Balances and sending funds.

### CODE REVIEW

**Review by FelixNicolaeBucsa — COMMENTED** [review]
@devjsc is the "copy" to be added here as well?

File: pages/guides/CosmPy/EstablishingNodeConnection.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Connecting to a blockchain 
+
+After successfully installing CosmPy, you can begin interacting with Cosmos-based blockchains by following these steps.
```
Comment (devjsc) [inline pages/guides/CosmPy/EstablishingNodeConnection.md]: With CosmPy installed you can ...

File: pages/guides/CosmPy/EstablishingNodeConnection.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+1. Start by setting up a new Python project where you'll be using the CosmPy library.
```
Comment (devjsc) [inline pages/guides/CosmPy/EstablishingNodeConnection.md]: intro sentence, then code, then what the code does imo

File: pages/guides/CosmPy/Install.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
+# Getting Started 
+
+Welcome to CosmPy, the Python-based framework that will streamline your interactions with Cosmos SDK based blockchains. Whether you are a seasoned blockchain developer or just starting your journey into the world of decentralized systems, CosmPy offers unparalleled simplicity with respect to basic and advanced interactions. Developed by Fetch.ai, this powerful Python library opens the gateway to seamless communication, making it easier than ever to explore the endless possibilities of decentralized applications.
```
Comment (devjsc) [inline pages/guides/CosmPy/Install.md]: these paragraphs feel a little puffy; why mention developed by fetch.ai?

reads more like marketing than docs.

File: pages/guides/CosmPy/QueryingAddressBalances.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
+# Querying Balances 
+
+Once you’ve established a connection to a network node you can use the `LegderClient` object to perform many useful operations such as querying balances. In particular the `LedgerClient` object allows you to:
```
Comment (devjsc) [inline pages/guides/CosmPy/QueryingAddressBalances.md]: Once you’ve established a connection to a network node [LINK to how to connect]...

File: pages/guides/CosmPy/TransactionBroadcast.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
+# Send Tokens
+
+CosmPy simplifies broadcasting transactions. After creating a wallet, you can send transactions by following the provided example. In it we first connect to the desired network then provide the desired destination address. The transaction is then broadcast using the `Ledger_client.send_tokens` function.
```
Comment (devjsc) [inline pages/guides/CosmPy/TransactionBroadcast.md]: anytime we assume the user has done something we should link to where they can learn how to do it

File: pages/guides/CosmPy/CreatingWallet.md
Comment (devjsc) [inline pages/guides/CosmPy/CreatingWallet.md:1]: empty file?

File: pages/guides/CosmPy/QueryingAddressBalances.md [hunk 6]
```diff
@@ -0,0 +1,33 @@
+# Querying Balances 
+
+Once you’ve established a connection to a network node [Guide on connecting](/Users/nikolaydimitrov/FetchProjects/docs_v2/pages/guides/CosmPy/EstablishingNodeConnection.md) you can use the `LegderClient` object to perform many useful operations such as querying balances. In particular the `LedgerClient` object allows you to:
```
Comment (devjsc) [inline pages/guides/CosmPy/QueryingAddressBalances.md]: your path to the guide is local?

File: pages/guides/CosmPy/QueryingAddressBalances.md [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+
+from cosmpy.aerial.client import LedgerClient, NetworkConfig
+
+ledger_client = LedgerClient(NetworkConfig.fetch_mainnet())
+
+address: str = 'fetch12q5gw9l9d0yyq2th77x6pjsesczpsly8h5089x'
+balances = ledger_client.query_bank_all_balances(address)
+print(balances)
+```
+Querying all balances associated with a particular address returns a `List` of `Coin` objects that contain amount and denom variables that correspond to all the funds held at the address and their denominations. This list includes all natively defined coins along with any tokens transferred using the inter-blockchain communication [IBC](https://ibcprotocol.org/) protocol. Once successful, you'll see the following input in the terminal.
```
Comment (devjsc) [inline pages/guides/CosmPy/QueryingAddressBalances.md]: links need the upright arrow emoji

**Review by devjsc — APPROVED** [review]

**Pull request closed without merging**